pkgs/by-name/li/libresample/package.nix

```diff
@@ -37,5 +67,6 @@ stdenv.mkDerivation (finalAttrs: {
       lib.maintainers.sander
       lib.maintainers.emily
     ];
+    mainProgram = "resample-sndfile";
```

Comment:
Wrong commit

Reply:
No, it’s the correct commit. The tools were not being built before.